config/v1/types_apiserver.go

```diff
@@ -39,6 +39,9 @@ type APIServerSpec struct {
 	// The values are regular expressions that correspond to the Golang regular expression language.
 	// +optional
 	AdditionalCORSAllowedOrigins []string `json:"additionalCORSAllowedOrigins,omitempty"`
+	// encryption allows the configuration of encryption types for encryption of resources at the datastore layer.
```

Comment:
I am not convinced they are right here in the generatic API server config. First of all we only implement them in kube-apiserver, the controller will be in kube-apiserver only (for now). Second froma  global config I assume – as a user – that they are applied to all API server (and all resource). Neither is the case.

Reply:
OAS must honor this config as well.  Otherwise OAuth tokens, routes, etc will not be protected.

Reply:
And service catalog? And user provider API servers? OLM?

I keep thinking that it is a bad API if it suggests that all API servers encrypt everything.

Reply:
Btw, we always talked about ConfigMaps and Secrets. Nothing else. Please add the list of resource we need to the OEP.